packages/widget/src/state/route.ts

```diff
@@ -87,6 +87,7 @@ export const routeConfigAtom = atom<WidgetRouteConfig>({
     splitRoutes: true,
     evmSwaps: true,
   },
+  goFast: true
```

Comment:
Is this ok to add here as a default or should I remove @codingki ?

Reply:
it's ok